src/tests/ftest/util/ec_utils.py

```diff
@@ -38,29 +38,33 @@ def get_data_parity_number(log, oclass):
 
 
 def check_aggregation_status(pool, quick_check=True, attempt=20):
-    """EC Aggregation triggered status.
+    """Check whether aggregation has started, determined by an increase in free space.
+
     Args:
-        pool(object): pool object to get the query.
-        quick_check(bool): Return immediately when Aggregation starts for any storage type.
-        attempt(int): Number of attempts to do pool query at interval of 5 seconds.
+        pool (object): pool object to get the query.
+        quick_check (bool, optional): Return as soon as Aggregation starts for any storage type.
+        attempt (int, optional): Number of attempts to do pool query at interval of 5 seconds.
                       default is 20 attempts.
-    return:
-        result(dic): Storage Aggregation stats SCM/NVMe True/False.
+
+    Returns:
+        dict: Storage Aggregation stats SCM/NVMe True/False.
     """
     agg_status = {'scm': False, 'nvme': False}
     pool.connect()
-    initial_usage = pool.pool_percentage_used()
+    cur_free_space = pool.pool_percentage_used()
+    pool.log.info("initial pool_percentage during Aggregation = {}".format(cur_free_space))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string